### PR TITLE
Fix cubic spline interpolation in glTF importer

### LINKED ISSUE #14636: glTF import bug
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
Arch Linux AMD64, Godot v3.0.beta.build1

**Issue description:**
Imported a gltf model from blender. Expect to just work™
Actual result:

First time, importing on dir change, crashed with a stack trace. Was not running in debug so I can't really provide it here, sorry.
Second time, imported successfully (was there on load but without the .import file due to me deleting it) but animations scaled instead of moving as they should

**Steps to reproduce:**
Heirarchy just needs:
ROOT: Spatial
.gtlf dragged in directly as a child

Note, .glb version (gltf binary) also exhibits the same behaviour

**Link to minimal example project:**

Not the project, but the source .blend and the exported gltf. This works in isolation.
[navdemo.zip](https://github.com/godotengine/godot/files/1642884/navdemo.zip)
*Updated to include required .bin file*

Animation of correct behaviour (in blender):
out 1

Sidenote. Apologies for the really low quality PR. This is really inconsistent on my computer and it caused so many issues I just gave up on trying to record it.

### LINKED ISSUE #21001: glTF: Errors in translation, in the CUBICSPLINE interpolation
**Godot version:**

tip 9f8e2d5c5bb0ab56630768530025e9e36c4bd445

**OS/device including version:**

Linux

**Issue description:**

The orange Suzanne in godot shouldn't play crazy, but follow the blue one with linear interpolation.

gltf-translation

**Minimal reproduction project:**

Test
[all-anim-broken.zip](https://github.com/godotengine/godot/files/2286973/all-anim-broken.zip)

Blenderfile
[suzanne-anim-broken.zip](https://github.com/godotengine/godot/files/2286977/suzanne-anim-broken.zip)

Working model w. linear translation

[suzanne-anim-linear.zip](https://github.com/godotengine/godot/files/2286980/suzanne-anim-linear.zip)

screenshot from 2018-08-14 15-02-15

Not working model w. bezier translation 
[suzanne-anim-broken.zip](https://github.com/godotengine/godot/files/2286981/suzanne-anim-broken.zip)

screenshot from 2018-08-14 15-02-47

### PR DESCRIPTION
**Pull request by goodyttoor**
Should solve #21001 #14636

*Bugsquad edit:* Fixes #21001, fixes #14636.

### CODE REVIEW

File: editor/import/editor_scene_importer_gltf.cpp [hunk 1]
```diff
@@ -1876,9 +1876,9 @@ T EditorSceneImporterGLTF::_interpolate_track(const Vector<float> &p_times, cons
 			float c = (p_time - p_times[idx]) / (p_times[idx + 1] - p_times[idx]);
 
 			T from = p_values[idx * 3 + 1];
-			T c1 = from + p_values[idx * 3 + 0];
-			T to = p_values[idx * 3 + 3];
-			T c2 = to + p_values[idx * 3 + 2];
+            T c1 = from + p_values[idx * 3 + 2];
```
Comment (akien-mga) [inline editor/import/editor_scene_importer_gltf.cpp]: You should use tabs for indentation.

Comment (goodyttoor) [reply editor/import/editor_scene_importer_gltf.cpp]: Oop sorry forget to format this

**Comment by goodyttoor** [comment]
Travis build fails strangely.

**Comment by capnm** [comment]
Good catch, fixes #21001, fixes #14636.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**